mite_selenium/__init__.py

```diff
@@ -162,10 +162,8 @@ def wait_for_element(self, locator, timeout=5):
                 f"Timed out trying to find element '{locator}' in the dom"
             ) from te
 
-    def switch_to_iframe(self, index):
-        self._remote.switch_to.frame(
-            self._remote.find_elements_by_tag_name("iframe")[index]
-        )
+    def switch_to_iframe(self, locator):
+        self._remote.switch_to.frame(self.wait_for_element(locator))
```

Comment:
this has different semantics...is there a reason not to use find_element?

Reply:
used wait_for_element because it was already there and it has added timeout functionality and underneath it does `find_element(*by)` so I don't have to expand it.

Reply:
ok, good point, I didn't think about that. Have made the change to use find_element